module/VuFind/src/VuFind/Db/Entity/UserList.php

```diff
@@ -165,6 +180,29 @@ public function getDescription(): ?string
         return $this->description;
     }
 
+    /**
+     * Get list type
+     *
+     * @return string
+     */
+    public function getType(): string
+    {
+        return $this->type ?? self::TYPE_DEFAULT;
```

Comment:
Do you need this default logic? I think by the way everything is set up, it should be impossible for `$this->type` to ever be null.

```suggestion
        return $this->type;
```